frontend/src/components/stats/ActiveAccountsByDate.tsx

```diff
@@ -3,6 +3,7 @@ import ReactEcharts from "echarts-for-react";
 import echarts from "echarts";
 
 import StatsApi, { AccountsByDate } from "../../libraries/explorer-wamp/stats";
+import { chartStyle } from "./TransactionsByDate";
```

Comment:
This is an unexpected place to find the styles. Can we set the styles on the parent component (I assume `stats/index.tsx` page) using CSS class and passing `className` down to the components?